test/API/H5_api_object_test.c

```diff
@@ -7375,6 +7365,23 @@ object_visit_noop_callback(hid_t o_id, const char *name, const H5O_info2_t *obje
     return 0;
 }
 
+/*
+ * Cleanup temporary test files
+ */
+static void
+cleanup_files(void)
+{
+    char filename[H5_API_TEST_FILENAME_MAX_LENGTH];
+
+    snprintf(filename, H5_API_TEST_FILENAME_MAX_LENGTH, "%s%s", test_path_prefix,
+             OBJECT_COPY_BETWEEN_FILES_TEST_FILE_NAME);
+    remove_test_file(NULL, filename);
```

Comment:
Could also just pass `test_path_prefix` to `remove_test_file` here to be even simpler.

Reply:
updated